tools/lib/logreader.py

```diff
@@ -13,7 +13,7 @@
 
 # this is an iterator itself, and uses private variables from LogReader
 class MultiLogIterator:
-  def __init__(self, log_paths, wraparound=False):
+  def __init__(self, log_paths, wraparound=False, sort_by_time=False):
```

Comment:
should we get rid of wraparound? think it was only used for unlogger

Reply:
Sounds good to me!